kernel_crawler/oracle.py

```diff
@@ -5,7 +5,7 @@
 class OracleRepository(rpm.RpmRepository):
     @classmethod
     def kernel_package_query(cls):
-        return '''(name IN ('kernel', 'kernel-devel', 'kernel-uek', 'kernel-uek-devel') AND arch = 'x86_64')'''
+        return '''(name IN ('kernel', 'kernel-devel', 'kernel-uek', 'kernel-uek-devel'))'''
```

Comment:
I'm also removing this arch-specific bit of the query here. Since the user can pass in the arch, I think we want to keep this arch-agnostic.